tests/cli/ursula/test_stakeholder_and_ursula.py

```diff
@@ -173,6 +173,38 @@ def test_staker_divide_stakes(click_runner,
     assert str(NU(token_economics.minimum_allowed_locked, 'NuNit').to_tokens()) in result.output
 
 
+def test_stake_prolong(click_runner,
+                       testerchain,
+                       test_registry,
+                       manual_staker,
+                       manual_worker,
+                       stakeholder_configuration_file_location):
+
+    prolong_args = ('stake', 'prolong',
+                    '--config-file', stakeholder_configuration_file_location,
+                    '--index', 0,
+                    '--lock-periods', 1,
```

Comment:
For variety, should we use a larger value than 1? 1 was already tested in the `test_staking_escrow_agent` prolong test.

Reply:
Yeah, this makes sense.

Reply:
The functionality of the contract is already tested at a lower layer, so I'm just showing that the higher layer has the desired state change and the command executes successfully.